scalableminds/webknossos
# Allow experience domains creation in task creation

This PR makes it possible to add new experience domains while creating tasks.

### URL of deployed dev instance (used for testing):
- https://allowexpdomaincreation.webknossos.xyz

### Steps to test:
- Open the task creation view
- Try to select an experience domain, that did not exist before.
- Also, open the admin's users view and try to change the experience domain of a user. It should work exactly as before.

### Issues:
- fixes #4073

------
